kolibri_explore_plugin/assets/src/views/SearchPage.vue

```diff
@@ -250,6 +256,18 @@
           params: { channel_id: channelId },
         });
       },
+      searchLastChannels(store, query, kind) {
```

Comment:
This all makes sense to me, but the name of this function doesn't work for me. I'd love if it conveyed a bit more what it's doing. Maybe `searchChannelsOnce`?

Reply:
Yeah `searchChannelsOnce` is much better. I'm not happy with the original `searchChannels` which is actually searching content per kind *in* channels. But I am preventing doing renames at this point.